code/game/objects/items/tools/kitchen_tools.dm

```diff
@@ -135,6 +135,44 @@
 
 	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 
+/*
+ * Plastic Pizza Cutter
+ */
+/obj/item/tool/kitchen/pizzacutter
+	name = "pizza cutter"
+	icon_state = "plasticpizzacutter"
+	desc = "A circular blade using for cutting pizzas. This one has a cheap plastic handle."
```

Comment:
I also think used is better than using here, using doesn't make much sense grammar wise.